pkg/cmd/roachtest/tests/restore.go

```diff
@@ -299,7 +300,7 @@ func registerRestore(r registry.Registry) {
 			hardware: makeHardwareSpecs(hardwareSpecs{
 				nodes: 9,
 				zones: []string{"us-east-2b", "us-west-2b", "eu-west-1b"}}), // These zones are AWS-specific.
-			backup:  makeBackupSpecs(backupSpecs{}),
+			backup:  makeBackupSpecs(backupSpecs{cloud: spec.AWS}),
```

Comment:
This fixes the noise from the cluster creation error: https://github.com/cockroachdb/cockroach/issues/78601#issuecomment-1561784414